valohai_yaml/objs/utils.py

```diff
@@ -74,6 +75,9 @@ def serialize_into(
     if flatten_dicts and isinstance(value, dict):
         value = list(value.values())
 
+    if issubclass(type(value), Enum) and not hasattr(value, "serialize"):
```

Comment:
I think this belongs in `_serialize_if_able`, below, and it should probably be just `if isinstance(value, Enum):`?

Reply:
I'm out of my comfort zone here, can you elaborate a bit how it would belong in the `_serialize_if_able`? :D I wouldn't necessarily bring enum value unwrapping into that function, as then the name should be `_serialize_if_able_or_unwrap_enum`, right?

Reply:
(We want enums to be serialized as their values, so I think the name is fine.)

Reply:
Gotcha, thanks!

Reply:
Now fixupped

Reply:
is a better place for it!

Reply:
or, well, now you can't define custom `serialize` for enums which was the original idea

but the place is better!